ckanext/saml2auth/views/saml2auth.py

```diff
@@ -185,8 +184,22 @@ def disable_default_login_register():
     return base.render(u'error_document_template.html', extra_vars), 403
 
 
+def unauthorised():
+    firstName = request.params.get('firstName', None)
+    lastName = request.params.get('lastName', None)
+    email = request.params.get('email', None)
+    extra_vars = {
+        u'code': [403],
+        u'name': u'Not Authorised', 
+        u'content': u' User {0} {1} with email {2} is not a member of any authenticated AD group'.format(firstName, lastName, email)
+    }
+    return base.render(u'error_document_template.html', extra_vars)
```

Comment:
@salsa-nathan I noticed in the above method disable_default_login_register that it used the template error_document_template.html so decided to use the same template to display the unauthorised message. Let me know if this fits the requirements or to create a new template?